### PR TITLE
url where recording originally happened can now be opened to a new tab on click

### LINKED ISSUE #1153: [Devtools] Add a way for a user to open the recording target URL in a new tab
There should be a way within Replay for the user to click on a link that opens up the recording target's URL.

### PR DESCRIPTION
**Pull request by karina-y**
fixes #1153

### CODE REVIEW

**Comment by karina-y** [comment]
failed ci tests run locally:
stepping-03 passes
logpoint-02 passes
breakpoints-05 passes
console_warp-02 times out in local branch and master
object_preview-03 times out in local branch

File: src/ui/components/Dashboard/DashboardViewerContent.js [hunk 1]
```diff
@@ -107,8 +107,8 @@ function DashboardViewerContentHeader({
         <input type="checkbox" onChange={handleHeaderCheckboxClick} checked={selectedIds.length} />
       </th>
       <th>PREVIEW</th>
-      <th>TITLE</th>
       <th>PAGE TITLE</th>
+      <th>REPLAY</th>
```
Comment (jaril) [inline src/ui/components/Dashboard/DashboardViewerContent.js:111]: Let's keep the same column labels here for now until we decide what to do with titles.

File: src/ui/components/Dashboard/RecordingItem/RecordingListItem.css [hunk 2]
```diff
@@ -231,3 +231,7 @@
   background: var(--theme-body-color);
   margin: auto;
 }
+
+.page-url:hover {
+  color: var(--dark-blue) !important;
```
Comment (jaril) [inline src/ui/components/Dashboard/RecordingItem/RecordingListItem.css:236]: Can we add `text-decoration: underline` here just to really drive the point home?

**Review by jaril — APPROVED** [review]

File: src/ui/components/Dashboard/DashboardViewerContent.js (hunk 1 above)
Comment (karina-y) [reply src/ui/components/Dashboard/DashboardViewerContent.js:111]: oh my bad forgot to undo that one, fixing now

File: src/ui/components/Dashboard/RecordingItem/RecordingListItem.css (hunk 2 above)
Comment (karina-y) [reply src/ui/components/Dashboard/RecordingItem/RecordingListItem.css:236]: sure

**Comment by jasonLaster** [comment]
Looks good!

**Comment by karina-y** [comment]
also screenshots for record!
before:
image


after:
image

**Pull request merged**